Source/DafnyLanguageServer/Language/Symbols/DafnyLangSymbolResolver.cs

```diff
@@ -150,9 +154,43 @@ private class SymbolDeclarationResolver {
           cancellationToken.ThrowIfCancellationRequested();
           functionSymbol.Parameters.Add(ProcessFormal(scope, parameter));
         }
+        if (function.Result != null) {
+          functionSymbol.Parameters.Add(ProcessFormal(scope, function.Result));
+        }
+        ScopeSymbol? ExpressionHandler(Expression? expression) =>
+          expression == null
+            ? null
+            : ProcessExpression(new ScopeSymbol(functionSymbol, functionSymbol.Declaration), expression);
+        functionSymbol.Body = ExpressionHandler(function.Body);
+        functionSymbol.Ensures = ProcessListAttributedExpressions(function.Ens, ExpressionHandler);
```

Comment:
For the sake of consistency, I'd opt to use this pattern instead:
```cs
functionSymbol.Body.AddRange(ExpressionHandler(function.Body));
```

Reply:
At this point, `FunctionSymbol.Body` is `null`, and there is no method `AddRange`. What did you mean exactly?